optaplanner-persistence/optaplanner-persistence-jpa/src/test/java/org/optaplanner/persistence/jpa/AbstractScoreJpaTest.java

```diff
@@ -74,7 +73,6 @@ public void tearDown() throws Exception {
         return id;
     }
 
-    @SafeVarargs
```

Comment:
Is this intentionally removed? I relates to a java warnings that `S... newScores` otherwises gives

Reply:
Oops, must have slipped during the refactor. I'll add it back.